src/components/image-config/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactElement, ReactNode } from 'react'
+import getValidatedImgChild from './get-validated-image-child'
+import MdxImg from '../mdx-img'
+import s from './image-config.module.css'
+
+/**
+ * Configure extra settings for images via MDX.
+ * Designed to wrap a single markdown image.
+ */
+export default function ImageConfig({
+  /** Hide borders (temporarily configurable while we remove baked-in borders from existing images) */
+  hideBorder = false,
+  /** A caption to display below the image. */
+  caption,
+  /** An MDX `img`, e.g. `![some alt text](path/to/img.jpg)` */
+  children,
+}: {
```

Comment:
I'm not sure if this was brought up before but could we extract this into an `interface` definition above the component? Just seems a bit easier to read imo :)

Reply:
👍  Ah yep, great call - did for `Image`, now done for `ImageConfig` too ✅